vignettes/Spectra.Rmd

```diff
@@ -1017,6 +1017,11 @@ could be used instead. The default backends are:
 - `MsBackendMzR`: this backend keeps only general spectra variables in memory
   and relies on the `r Biocpkg("mzR")` package to read mass peaks (m/z and
   intensity values) from the original MS files on-demand.
+  
+- `r Biocpkg("MsBackendRawFileReader")`: implements a backend for reading
```

Comment:
I would suggest to move this paragraph a bit down were other (external) packages providing backends are listed (line 1044 onwards where e.g. `MsBackendMgf` etc are listed).

Also, maybe I would remove the "Orbitrap" and clearly speficy that your backend support importing data directly from the **raw** data files for Thermo MS Instruments.

Reply:
yes; fine with me. only one thing. can you replace `Thermo` with `Thermo Fisher Scientific`.